### DISCUSSION TITLE
custom background color

### DISCUSSION THREAD

**Discussion by jenninep1** [add2cal/add-to-calendar-button#475] [Q&A] [answered]
Can someone help with how to change the background and border? I have tried this css among many other classes and ID variations to no avail. hoping for a solution. thanks in advance
.atcb-button {
background-color: #ffab01 !important;
border: none !important;
}

**Comment by jekuer** [comment, marked as answer]
Hi @jenninep1 ,

Your approach will not work, because the button is protected by its own shadowDOM.
You can adjust those values via the styleLight and styleDark options (see https://add-to-calendar-button.com/de/configuration#stylelight).
In your case, asuming you are using the default button and only light mode, you could use `styleLight="--btn-background: #ffab01;--btn-border:none;"`

Best,
Jens

**Comment by jenninep1** [comment]
Thank you so much for your response.  Is there also an option for the hover color? It is what right now which doesn’t go with the websites color palette. 
https://warriorsofstone.com/￼

**Reply by jekuer** [reply]
you can override all variables you can find at https://github.com/add2cal/add-to-calendar-button/blob/main/assets/css/atcb.css - the ones starting with "--".
